bridges/VarietyBridge.php

```diff
@@ -0,0 +1,25 @@
+<?php
+class VarietyBridge extends FeedExpander {
+
+	const MAINTAINER = 'IceWreck';
+	const NAME = 'Variety Bridge';
+	const URI = 'https://variety.com';
+	const CACHE_TIMEOUT = 3600;
+	const DESCRIPTION = 'RSS feed for Variety';
+
+	public function collectData(){
+		$this->collectExpandableDatas('http://feeds.feedburner.com/variety/headlines', 15);
+	}
+
+	protected function parseItem($newsItem){
+		$item = parent::parseItem($newsItem);
+		// $articlePage gets the entire page's contents
+		$articlePage = getSimpleHTMLDOM($newsItem->link);
```

Comment:
Thanks for your contribution :+1: 

Here is a suggestion for how to remove script tags in the output.

```suggestion
		$articlePage = getSimpleHTMLDOM($newsItem->link);
		
		foreach($articlePage->find('.pmc-contextual-player') as $player) {
			$player->remove();
		}
		
```

Please note that script tags look ugly in any format, not just HTML because RSS-Bridge generally returns script tags as literal text.

Reply:
Thanks. To be honest, I forgot about this PR since I installed the bridge on my own instance.

I've fixed it now in the latest commit.